projects/dotcms-field-elements/src/components/dot-textfield/dot-texfield.e2e.ts

```diff
@@ -6,135 +6,282 @@ describe('dot-textfield', () => {
+            });
+            it('should set placeholder correctly with invalid value', async () => {
+                element.setProperty('placeholder', { test: 'hi' });
+                await page.waitForChanges();
+                expect(await input.getProperty('placeholder')).toBe('[object Object]');
+            });
+        });
 
-    it('should clear value, set pristine and untouched  when input set reset', async () => {
-        await element.callMethod('reset');
-        await page.waitForChanges();
+        describe('hint', () => {
+            it('should set hint correctly', async () => {
+                element.setProperty('hint', 'Test');
+                await page.waitForChanges();
+                expect((await getHint()).innerHTML).toBe('Test');
+            });
 
-        expect(element.classList.contains('dot-pristine')).toBe(true);
-        expect(element.classList.contains('dot-untouched')).toBe(true);
-        expect(element.classList.contains('dot-invalid')).toBe(true);
-        expect(await input.getProperty('value')).toBe('');
-    });
+            it('should not render hint', async () => {
+                expect(await getHint()).toBeNull();
+            });
 
-    it('should mark as disabled when prop is present', async () => {
-        element.setProperty('disabled', true);
-        await page.waitForChanges();
-        expect(await input.getProperty('disabled')).toBe(true);
-    });
+            it('should not render  hint with invalid value', async () => {
+                element.setProperty('hint', { test: 'hint' });
+                await page.waitForChanges();
+                expect(await getHint()).toBeNull();
+            });
+        });
 
-    it('should mark as required when prop is present', async () => {
-        expect(await input.getProperty('required')).toBe(true);
-    });
+        describe('required', () => {
+            it('should render required attribute', async () => {
+                element.setProperty('required', 'true');
+                await page.waitForChanges();
+                expect(await input.getProperty('required')).toBe(true);
+            });
 
-    it('should set the default value of regexCheck when the Regular Expression is not valid', async () => {
-        element.setProperty('regexCheck', '[^(<[.\\n]+>)]*l');
-        await page.waitForChanges();
-        expect(await input.getProperty('regexCheck')).toBeUndefined();
-    });
+            it('should not render required attribute', async () => {
+                element.setProperty('required', 'false');
+                await page.waitForChanges();
+                expect(await input.getProperty('required')).toBe(false);
+            });
 
-    describe('emit events', () => {
-        it('should mark as touched when onblur', async () => {
-            await input.triggerEvent('blur');
-            await page.waitForChanges();
+            it('should render required attribute with invalid value', async () => {
```

Comment:
```suggestion
            it('should render required attribute with truthy value', async () => {
```

Reply:
Should we also check falsy value?

Reply:
is done in the previous test, https://github.com/dotCMS/core-web/pull/1032/files#diff-5fdc3173364322f489e7fb5fd2408fafR160